### ISSUE TITLE
FAST Farm T1 initialize Error

### ISSUE THREAD

**Issue by godsentaustine** [OpenFAST/openfast#2027]
i am running FAST Farm with the OC3 Hywind configuration. and I'm getting this error after updating the ''--- AMBIENT WIND: INFLOWWIND MODULE'' parameters. I've tried to figure out where this is coming from. any help will be appreciated as i'm quite new using FAST Farm.

![image](https://github.com/OpenFAST/openfast/assets/126730108/8cc405af-c3ef-404a-bf65-655894c31a93)

![image](https://github.com/OpenFAST/openfast/assets/126730108/17b17ab1-ff82-42cd-9152-26262d755e60)

**Comment by jjonkman** [comment]
Dear @godsentaustine,

Similar questions were asked and answered in the following OpenFAST issues: https://github.com/OpenFAST/openfast/issues/1613 and https://github.com/OpenFAST/openfast/issues/1044.

Best regards,

**Comment by godsentaustine** [comment]
Dear @jjonkman ,
Thnaks for your reply. I've been able to follow the [python-toolbox](https://github.com/OpenFAST/python-toolbox/tree/main) to generate FAST.Farm input. (The Farm layout is a 3x2 and are 5D from each other). The error i was getting was still coming up not until i changed dr = cmax. that appeared to have resolved that error. 
The error below is what i'm currently encountering. Does this have to do with the number of wake planes? because i did try to increase it which did'nt work. Where can I resolve this?
![image](https://github.com/OpenFAST/openfast/assets/126730108/b1ef901e-a58d-4643-880c-8c83940b158a)

![image](https://github.com/OpenFAST/openfast/assets/126730108/e21f413c-9986-46f9-9738-be5b48365b4b)

![image](https://github.com/OpenFAST/openfast/assets/126730108/e90c7528-00b3-425d-91f6-3a5795fac8f7)

**Comment by godsentaustine** [comment]
The Y0_Low parameter sets the origin of the Y boundary to -80.0 m, and with NY_Low being 8 and dY_Low being 18.608 m, this gives a total coverage in the Y direction of only 8 * 18.608 m = 148.864 m, which doesn't seem sufficient to cover turbine #4's Y position (630 m). my thinking is i'd need to adjust Y0_Low and potentially NY_Low or dY_Low to ensure the low-resolution grid covers these turbines. Is this reasonable? since i'd be altering the calculations from the modelling guidance which i presume the toolbox follows.

**Comment by jjonkman** [comment]
I agree that your low-resolution domain is not sufficient for the wind farm you are simulating.  It sounds like you are reviewing the Python script rather than using it; is that correct?  I would suggest using the Python script, which generally follows the FAST.Farm modeling guidance.

Please note that the width of the low-resolution domain is actually ( `NY_Low` - 1 )*`dY_Low` because the nodes are at the extremities of the domain.

Best regards,

**Comment by godsentaustine** [comment]
dear @jjonkman 
Your reply has been helpful. I've finally got it to run successfully. I'm still quite new to OpenFAST. 
I am trying to implement different yaw settings in the ElastoDyn (initial conditions) to see how it will affect generated power. I have the errors on the snapshot below when i make changes to NacYaw. What could be causing these? I have no errors when i set NacYaw to 0.

![image](https://github.com/OpenFAST/openfast/assets/126730108/e413ea4b-153d-49e3-94fd-9856fdde0548)

Here is how my initial conditions looks like. I am simulating 12m/s at Hub Height 90m
 
![image](https://github.com/OpenFAST/openfast/assets/126730108/8dc1f420-ca35-4672-9be3-169af31fdedc)

**Comment by jjonkman** [comment]
Dear @godsentaustine,

Do you have the nacelle-yaw degree of freedom enabled in ElastoDyn (`YawDOF` = TRUE)?  If so, I would guess you are facing a similar issue to the one posted on our forum, where I offer a simple solution--see my post dated Mar 10, 2016 here: https://forums.nrel.gov/t/wind-farm-modeling-by-using-fast-and-turbsim/326/10?u=jason.jonkman.

Best regards,

**Comment by godsentaustine** [comment]
Dear @jjonkman 
I have taken a look up at your post as advised, that did solve the issue setting _NacYaw_ = _YawNeut_ 
Below is the warning i receive about high angle of attack during simulation. will this affect the accuracy of the results?  

*************************************************************
Running ServoDyn.
Running ServoDyn Interface for Bladed Controllers (using Intel Visual Fortran for Windows).
Using legacy Bladed DLL interface.

Farm_Initialize:Farm_ValidateInput:OutFmt produces a column width of 10 instead of 20 characters.

 Time: 101 of 1500 seconds.  Estimated final completion at 18:30:56.

T1:FARM_UpdateStates:FWrap_Increment:FAST_Solution:CalcOutputs_And_SolveForInputs:SolveOption2:AD_
CalcOutput:RotCalcOutput:BEMT_CalcOutput(node 5, blade
3):UA_CalcOutput:UA_BlendSteady:Temporarily turning off UA due to high angle of attack or low
relative velocity. This warning will not be repeated though the condition may persist.
 Time: 164 of 1500 seconds.  Estimated final completion at 18:35:25.

T5:FARM_UpdateStates:FWrap_Increment:FAST_Solution:CalcOutputs_And_SolveForInputs:SolveOption2:AD_
CalcOutput:RotCalcOutput:BEMT_CalcOutput(node 5, blade
2):UA_CalcOutput:UA_BlendSteady:Temporarily turning off UA due to high angle of attack or low
relative velocity. This warning will not be repeated though the condition may persist.
 Time: 307 of 1500 seconds.  Estimated final completion at 18:31:59.

T4:FARM_UpdateStates:FWrap_Increment:FAST_Solution:CalcOutputs_And_SolveForInputs:SolveOption2:AD_
CalcOutput:RotCalcOutput:BEMT_CalcOutput(node 5, blade
1):UA_CalcOutput:UA_BlendSteady:Temporarily turning off UA due to high angle of attack or low
relative velocity. This warning will not be repeated though the condition may persist.
 Time: 595 of 1500 seconds.  Estimated final completion at 18:31:55.

T3:FARM_UpdateStates:FWrap_Increment:FAST_Solution:CalcOutputs_And_SolveForInputs:SolveOption2:AD_
CalcOutput:RotCalcOutput:BEMT_CalcOutput(node 5, blade
3):UA_CalcOutput:UA_BlendSteady:Temporarily turning off UA due to high angle of attack or low
relative velocity. This warning will not be repeated though the condition may persist.
 Time: 1352 of 1500 seconds.  Estimated final completion at 18:37:19.

T6:FARM_UpdateStates:FWrap_Increment:FAST_Solution:CalcOutputs_And_SolveForInputs:SolveOption2:AD_
CalcOutput:RotCalcOutput:BEMT_CalcOutput(node 5, blade
3):UA_CalcOutput:UA_BlendSteady:Temporarily turning off UA due to high angle of attack or low
relative velocity. This warning will not be repeated though the condition may persist.


 Total Real Time:       72.902 minutes
 Total CPU Time:        46.077 minutes
 Simulation CPU Time:   45.994 minutes
 Simulated Time:        24.99 minutes
 Time Ratio (Sim/CPU):  0.54333

 FAST.Farm terminated normally.

***************************************************************

Second question is, what could be the reason i get the errors on the snap shot below when i try to change the Hs and Tp value?

![image](https://github.com/OpenFAST/openfast/assets/126730108/1ec4a791-2b9f-4643-9fdd-edf4d8739222)

the values on the snapshot above seems to work fine. changing them to like Hs = 17.9 and Tp - 14.4 gives the errors below

![image](https://github.com/OpenFAST/openfast/assets/126730108/b77a1828-450e-46cd-aaea-9faf4e753377)

**Comment by godsentaustine** [comment]
The Rotor speed, windspeed and blade pitch looks like this up until failure

![image](https://github.com/OpenFAST/openfast/assets/126730108/fab37681-1ac2-4877-b666-4cf85f926d50)

**Comment by jjonkman** [comment]
Dear @godsentaustine,

Regarding your first question, you will receive this warning if the angle of attack exceeds the limits of applicability of the unsteady airfoil aerodynamic model.  I would not be particularly concerned when receiving this warning at inboard stations of the blade if other aspects of the simulation are as expected.  It looks like the simulation terminated normally anyway.

Regarding your second question, I'm not sure.  The sea state you are setting is very severe, which I would guess is resulting in excessive motion of the FOWT that results in an unrealistic thrust coefficient (at least temporarily).  But I'd have to to see other response variables (such as floater motions) to comment.

Best regards,

**Comment by godsentaustine** [comment]
Dear @jjonkman 
indeed the sea state is very severe the motions appear to be sharp . I have attempted another simulation and its a different error but i feel could be the same issue?
 
![image](https://github.com/OpenFAST/openfast/assets/126730108/9f7167c4-03f1-4e50-9693-912a5286e493)

![image](https://github.com/OpenFAST/openfast/assets/126730108/6bf9f55f-7722-4948-80ac-4cbc80d8aee0)

**Comment by jjonkman** [comment]
Dear @godsentaustine,

Indeed, now it appears that the OpenFAST model has become unstable.  Is this simulation also excited by a very severe sea state?

Best regards,

**Comment by godsentaustine** [comment]
yes it is
